client/src/containers/AppContainer/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import App from 'src/components/App'
+import api from './requests'
+
+class AppContainer extends Component {
```

Comment:
`AppContainer ` is a part of `containers` folder, so can be used without `Container` postfix.

The same pattern as for `components/`

Reply:
@AleksSenkou actually there'll be a conflict with `App` component if I rename it

Reply:
@DDKatch yeah, what about file naming? can we not use `Container` there?

Reply:
@AleksSenkou of course 😉